File: COData-WEB-Desarrollo-isai/BackEnd-VS-C#/COData-Web_BackEnd/COData-Web_BackEnd/Models/HistorialReportes.cs
namespace COData_Web_BackEnd.Models
{
    public class HistorialReportes
    {
        public int Historial_id { get; set; }
        public int Reporte_id { get; set; }
        public int Usuario_id { get; set; }
        public string Campo_Modificado { get; set; } = string.Empty;
        public string Valor_Anterior { get; set; } = string.Empty;
        public string Valor_Nuevo { get; set; } = string.Empty;
        public DateTime Fecha { get; set; }
        // Relación
        public Reportes Reporte { get; set; } = new();
        public Usuario Usuario { get; set; } = new Usuario();
    }
}

File: COData-WEB-Desarrollo-isai/FrontEnd-VS-Code/COData-Angular/src/app/models/historial-reportes.ts
import { Usuario } from './usuario';
import { Reportes } from './reportes';

export interface HistorialReportes {
  historial_id: number;
  reporte_id: number;
  usuario_id: number;
  campo_modificado: string;
  valor_anterior: string;
  valor_nuevo: string;
  fecha: Date;
  reporte?: Reportes;
  usuario?: Usuario;
}